sycamore/tests/unit/evaluation/test_pipeline.py

```diff
@@ -0,0 +1,43 @@
+import datasets
+
+import sycamore
+from sycamore.evaluation.datasets import EvaluationDataSetReader
+from sycamore.evaluation.pipeline import EvaluationPipeline
+from sycamore.transforms.query import OpenSearchQueryExecutor
+
+
+class TestEvaluationPipeline:
+    INDEX = "toyindex"
+
+    OS_CLIENT_ARGS = {
+        "hosts": [{"host": "localhost", "port": 9200}],
+        "http_compress": True,
+        "http_auth": ("admin", "admin"),
+        "use_ssl": True,
+        "verify_certs": False,
+        "ssl_assert_hostname": False,
+        "ssl_show_warn": False,
+        "timeout": 120,
+    }
+
+    OS_CONFIG = {
+        "size": 10,
+        "neural_search_k": 100,
+        "embedding_model_id": "pZabXI0BwnxcM6YnF2QF",
+        "search_pipeline": "hybrid_rag_pipeline",
+        "llm": "gpt-3.5-turbo",
+        "context_window": "5",
+    }
+
+    def test_hf(self):
+        context = sycamore.init()
+        reader = EvaluationDataSetReader(context)
+        mapping = {"question": "question", "answer": "answer", "document_url": "doc_link"}
+        hf_dataset = datasets.load_dataset("PatronusAI/financebench", split=datasets.Split.TRAIN)
```

Comment:
shouldn't this be the test/validation split for evaluation? also in the other test

Reply:
This is just ensuring we can process a dataset. This one is train since it's a sample dataset that only exposes train